addons/api/mirage/config.js

```diff
@@ -73,7 +73,11 @@ function routes() {
           return scope.scope ? scope.scope.id === scope_id : false;
         });
       }
-      return resultSet.filter(makeBooleanFilter(filter));
+      const filteredResultSet = resultSet.filter(makeBooleanFilter(filter));
+      // add a list_token to the response to simulate pagination
+      // without having list_token, the unsupported controller modal
+      // will show when running with mirage
+      return { items: filteredResultSet.models, list_token: '12345' };
```

Comment:
Should this change be made for all routes?

Also can we add some acceptance tests now for this PR as well?

Reply:
If we don't make this change here, the tests for desktop will always show the modal. I was thinking we make the change here then update the response on in the test cases that need to test the modal. Happy to discuss alternative methods though.

Reply:
Yup, I meant we should probably make it so all routes return this data and think about how to accomplish that but we can tackle that in the next PR